migrations/frontend/1528395841_add_repo_table_policy.up.sql

```diff
@@ -0,0 +1,47 @@
+BEGIN;
+
+-- When row-level security is enabled, the table immediately "fails closed"
+-- for all roles other than the table owner. The "sg_repo_access_policy" then
+-- dictates the filtering mechanism used to decide what rows can be seen or
+-- updated by the specified role(s).
+--
+-- The USING clause requires two LOCAL variables to be set:
+--     1. rls.user_id: the effective ID for the user making the request
+--     2. rls.permission: the permission that the user needs (e.g. "write")
```

Comment:
```suggestion
-- The USING clause requires three LOCAL variables to be set:
--     1. rls.use_permissions_user_mapping: the switch to turn on permissions uesr mapping
--     2. rls.user_id: the effective ID for the user making the request
--     3. rls.permission: the permission that the user needs (e.g. "read")
```

nit: `"read"` is the only possible value so far, I figure using "write" would actually confuse our future selves.